src/integration-test/java/com/mongodb/jdbc/integration/testharness/TestGenerator.java

```diff
@@ -0,0 +1,136 @@
+package com.mongodb.jdbc.integration.testharness;
+
+import com.mongodb.jdbc.integration.MongoIntegrationTest;
+import com.mongodb.jdbc.integration.MongoSQLIntegrationTest;
+import com.mongodb.jdbc.integration.testharness.models.Tests;
+import java.io.File;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.lang.reflect.InvocationTargetException;
+import java.sql.Connection;
+import java.sql.ResultSet;
+import java.sql.ResultSetMetaData;
+import java.sql.SQLException;
+import java.text.SimpleDateFormat;
+import java.util.ArrayList;
+import java.util.Date;
+import java.util.LinkedHashMap;
+import java.util.List;
+import java.util.Map;
+import org.yaml.snakeyaml.DumperOptions;
+import org.yaml.snakeyaml.Yaml;
+
+public class TestGenerator {
+    private static final String generatedTestDirectory = "resources/generated_test";
```

Comment:
```suggestion
    private static final String GENERATED_TEST_DIR = "resources/generated_test";
```
same here re: idiomatic `static final` field names